recipes/utfcpp/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "utfcpp" %}
+{% set version = "2.3.5" %}
+
+package:
+   name: {{ name|lower }}
+   version: {{ version }}
+
+source:
+   git_rev: {{ "v" ~ version }}
+   git_url: https://github.com/nemtrif/utfcpp.git
```

Comment:
Hi, thank you for the review. 
Yes I can do that, but please explain, why using a git tag is not as good as using a tarball. Esp. as long as git urls are supported natively by conda build. - thanks.